pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -311,6 +317,73 @@ func initProxy(args []string) (*model.Proxy, error) {
 	return proxy, nil
 }
 
+func applyExcludeInterfaces(ifaces []string) []string {
+	// Get list of excluded interfaces from pod annotation
+	annotations, err := bootstrap.ReadPodAnnotations("")
+	if err != nil {
+		log.Infof("Failed to read pod annotations to find exluded interfaces. Continuing without exclusions. Proxy IPAddresses: %v", ifaces)
+		return ifaces
+	}
+	value, ok := annotations[annotation.SidecarTrafficExcludeInterfaces.Name]
```

Comment:
Isn't the annotation translated into an env variable ( it is used in init container ). 

I don't know if pod annotations is the right input here - exclude interface is used quite frequently on VMs and serverless use cases, and annotations are more complicated (  I don't think we want to translate the env variables back into annotations ).

Reply:
Sure, I will add a TODO... Excludeinterfaces is already handled in istio-iptable, I wanted to handle the same without having to ask the users to configure excludeinterfaces at multiple places.

Reply:
Added a commit. Setting this to resolved. Please let me know if any further concerns.